x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/transform/transform.test.ts

```diff
@@ -11,6 +11,7 @@ jest.mock('../../packages/get', () => {
 jest.mock('./common', () => {
   return {
     getAsset: jest.fn(),
+    getLogger: jest.fn(),
```

Comment:
Please drop this and add something like https://github.com/elastic/kibana/blob/e48a5672c07eed9e1e15c032b8950013e6b8f95a/x-pack/plugins/ingest_manager/server/routes/package_policy/handlers.test.ts#L68 in a before/beforeEach